### PR TITLE
Issue templates: remove the requirement for issue title prefixes #6631

### LINKED ISSUE #6631: Issue templates: remove the requirement for issue title prefixes
Requiring a prefix e.g. `Bug report:` is redundant as there are labels to indicate the type of an issue (e.g. `c.Bug`). Instead of using part of the issue title to show redundant info, it's more profitable to use prefixes (optionally) to indicate the scope of the issue. e.g. `Issue templates:`

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #6631 

**Outline of Solution**

As per lined out in the issue thread.

### CODE REVIEW

File: .github/template-featureRequest.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-**Issue Title:** Feature: [your request in a line] OR Enhancement: [your suggestion in a line]
+**Issue Title:** [your feature/enhancement request in a line]
```
Comment (damithc) [inline .github/template-featureRequest.md]: `in a line` is redundant?

**Comment by wkurniawan07** [comment]
I feel like `c.Bug`, `c.Story`, and `c.Enhancement` should be strictly kept to user-facing features, but that would make `c.Task` too broad.
Can I suggest one extra label for build-related or workflow-related improvement or addition (like this issue, and the IntelliJ support issue)? The label can be `c.DevOps` or something similar.
So `c.Task` will be other stuffs like refactoring, adding tests, updating docs, or removing legacy code.

**Comment by wkurniawan07** [comment]
Also, @samsontmr made the following suggestion (in the issue thread):

> We could also use the first letter of the category to represent it to cut down on character count.

It's not exactly future-proof since we can't guarantee that we're not going to add more labels that will have clashes on the first letter, and it may not be immediately clear what each letter represents for those looking at the commit messages.

**Comment by damithc** [comment]
> I feel like c.Bug, c.Story, and c.Enhancement should be strictly kept to user-facing features, but that would make c.Task too broad.
> Can I suggest one extra label for build-related or workflow-related improvement or addition (like this issue, and the IntelliJ support issue)? The label can be c.DevOps or something similar.
> So c.Task will be other stuffs like refactoring, adding tests, updating docs, or removing legacy code.

Agree. Let's add `c.DevOps`



> I realize it will make a-DevOps seem like a duplicate. For this, I suggest decommissioning that label and move its classifications to other labels (some are new):
> 
> Dev docs -> a-Docs
> Static analysis -> a-CodeQuality
> Workflow and release management -> a-Workflow (new label)
> IDE support, CI, task automation -> a-Build (new label)

Sure, let's do that. Go ahead and add those labels.


> Also, @samsontmr made the following suggestion (in the issue thread):
> 
>  > We could also use the first letter of the category to represent it to cut down on character count.
>
> It's not exactly future-proof since we can't guarantee that we're not going to add more labels that will have clashes on the first letter, and it may not be immediately clear what each letter represents for those looking at the commit messages.

One letter may be too cryptic. If only we can find a shorter replacement for `Enhancement`. May be `c.Tweak`?

**Comment by wkurniawan07** [comment]
`c.Enhance`?

**Comment by damithc** [comment]
> c.Enhance?

A bit odd because others are nouns.

**Comment by wkurniawan07** [comment]
How about `c.Update`?

Or alternatively, we make a one-to-one mapping from `c.*` label to some keyword.

**Comment by damithc** [comment]
> How about c.Update?

Sounds like something was 'outdated' before. This is not always the case. Let's use `c.Enhance`

**Comment by wkurniawan07** [comment]
I think I'll be leaving the issue `c.*` label out from the merge commit for now; not meaningful unless we use it for some purpose like changelog.

**Pull request merged**